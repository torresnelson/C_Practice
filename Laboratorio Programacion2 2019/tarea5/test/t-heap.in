# prueba de tiempo heap
tiempo_heap 1000000
Fin
