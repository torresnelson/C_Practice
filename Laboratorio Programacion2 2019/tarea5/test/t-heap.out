1>#  prueba de tiempo heap.
2>
 prueba 1. 
 prueba 2. 
 prueba 3. 
 prueba 4. 
3>Fin.
